frontend/src/app/pages/browse/browse.component.html

```diff
@@ -13,14 +13,14 @@ <h2>Newly Added</h2>
                 <div style="margin: 0 auto 1rem auto;" *ngFor="let work of works">
                     <div class="work-container">
                         <!--Header-->
-                        <div class="work-header">
-                            <div style="display: inline-block;" [ngSwitch]="work.meta.rating">
+                        <div class="work-header">                            
+                            <ng-container [ngSwitch]="work.meta.rating">
```

Comment:
+1 to `<ng-container>`. It doesn't show up in the DOM, but is targetable by Angular directives.